macros/src/main/scala/me/rexim/morganey/meta/QuotationMacro.scala

```diff
@@ -0,0 +1,161 @@
+package me.rexim.morganey.meta
+
+import me.rexim.morganey.ast._
+
+import scala.collection.mutable.ArrayBuffer
+import scala.reflect.macros.whitebox.Context
+
+private[meta] class QuotationMacro(val c: Context) {
```

Comment:
I think that the macro library is an ordinary sbt project and it actually can be tested without involving our main project; we could call that our _unit_ tests. At the same time, we can and should also test the results of the macro-expansion, that will be form our _integration_ test layer.

Personally I think that it's preferable to include tests in the same PR as the main code. But it'd be okay to write them a bit later. Let @rexim to choose.